docs/advanced/debugging.md

```diff
@@ -5,122 +5,169 @@ order: 6
 
 # Debugging
 
-We've tried to make debugging in `urql` as seamless as possible by creating tools for users of `urql` and those creating their own exchanges.
+We've tried to make debugging in `urql` as seamless as possible by creating tools for users of `urql`
+and those creating their own exchanges.
 
-## Debug events
-
-Debug events are a means for seeing what's going on inside of `urql` exchanges. Before getting started, here are a few things to be aware of.
-
-### Anyone with client access can listen
-
-This includes the creator of an `urql` client and any of it's exchanges (although the latter is not advised).
-
-### Debug events are disabled in production
-
-Debug events are a fire-and-forget mechanism and should not be used as a means for messaging in your app.
+## Browser Devtools
```

Comment:
```suggestion
## Devtools
```

Can we leave this as "Devtools" - more granular usage details which aren't specific to Urql core (such as browser/native usage) will be in the Repo's README